knapsack_rspec_report.json

```diff
@@ -147,7 +147,6 @@
   "spec/controllers/test/telephony_controller_spec.rb": 0.038121855999999996,
   "spec/controllers/two_factor_authentication/backup_code_verification_controller_spec.rb": 0.686278922,
   "spec/controllers/two_factor_authentication/options_controller_spec.rb": 0.365245657,
-  "spec/controllers/two_factor_authentication/otp_expired_controller_spec.rb": 0.242342297,
```

Comment:
Question for my understanding: What does this line do?

Reply:
Knapsack helps parallelize tests evenly across build runners based on how long each test file takes to run, so the value here is the relative "weight" of the spec. Since the spec is being removed, we don't need to track it anymore.

Ideally this would happen semi-automatically, but I just happened to stumble across this in searching for any lingering references to "otp_expired".